resources/qml/RoomList.qml

```diff
@@ -229,6 +231,7 @@ Page {
                     width: avatarSize
                     url: avatarUrl.replace("mxc://", "image://MxcImage/")
                     displayName: roomName
+                    userid: roomMemberCount < 3 ?  directChatAvatarMxid : roomId
```

Comment:
iirc the userid is also used for the online indicator. You should probably make a separate property for the roomId

Reply:
Yeah, I was just too lazy to :P

Reply:
bad! ;p